velox/dwio/type/fbhive/HiveTypeParser.h

```diff
@@ -50,11 +50,10 @@ enum class TokenType {
   Number = 17,
   Identifier = 18,
   EndOfStream = 19,
-  ShortDecimal = 20,
-  LongDecimal = 21,
-  LeftRoundBracket = 22,
-  RightRoundBracket = 23,
-  MaxTokenType = 23
+  ShortDecimal,
```

Comment:
Removed the numbering as it is not necessary and every new type added requires updating values upto MaxTokenType. Here, MaxTokenType serves as the limit.